### ISSUE TITLE
Improved loss functions

### ISSUE THREAD

**Issue by JakeForsey** [JakeForsey/blindness-detection#37]
Handle the ordinal nature of the classification problem with loss functions.

 - [ ] NNRank
 - [ ] Kappa (maybe just use the sklearn implementation?)
